vdb/debug_computation.py

```diff
@@ -82,16 +87,19 @@ def apply_computation(cls, state, message, transaction_context):
                 opcode_fn = computation.get_opcode_fn(opcode)
 
                 pc_to_execute = max(0, computation.code.pc - 1)
-                computation.logger.trace(
-                    "OPCODE: 0x%x (%s) | pc: %s",
-                    opcode,
-                    opcode_fn.mnemonic,
-                    pc_to_execute,
-                )
+                if cls.trace:
+                    print(
+                        "NEXT OPCODE: 0x%x (%s) | pc: %s..%s" %
+                        (opcode,
+                        opcode_fn.mnemonic,
+                        cls.pc,
+                        pc_to_execute)
+                    )
+                cls.pc = pc_to_execute
 
                 is_breakpoint, line_no = cls.is_breakpoint(pc_to_execute, continue_line_nos)
```

Comment:
Does this skip until the next line, or the next PC ? I'd imagine we want to skip until the next line.

Reply:
The next pc. I guess both are useful, since a line could execute many instuctions. But to skip to the next line I could always just insert a `vdb` before every line :)

Reply:
Maybe it should be renamed `stepi`? gdb has
```
next -- Step program
nexti -- Step one instruction
...
start -- Start the debugged program stopping at the beginning of the main procedure
starti -- Start the debugged program stopping at the first instruction
step -- Step program until it reaches a different source line
stepi -- Step one instruction exactly
```

Reply:
Yes I agree without i becomes line stepping :)